src/usr/local/www/services_dhcp_edit.php

```diff
@@ -385,8 +385,9 @@ function staticmaps_sort($ifgui) {
 
 // Get our MAC address
 $ip = $_SERVER['REMOTE_ADDR'];
-$mymac = `/usr/sbin/arp -an | grep '('{$ip}')' | cut -d" " -f4`;
-$mymac = str_replace("\n", "", $mymac);
+
+unset($mymac);
```

Comment:
unset is unnecessary here since you are attributing a value to the same variable in the next line